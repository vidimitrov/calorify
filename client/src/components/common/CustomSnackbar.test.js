import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import CustomSnackbar from './CustomSnackbar';

it('Common/CustomSnackbar component renders correctly', () => {
  const tree = renderer.create(
    <CustomSnackbar
      onClose={() => { }}
      variant="success"
      message="Meal created successfully"
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
