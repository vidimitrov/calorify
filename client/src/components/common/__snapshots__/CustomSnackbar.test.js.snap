// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Common/CustomSnackbar component renders correctly 1`] = `
<div
  aria-describedby="client-snackbar"
  className="MuiTypography-root-38 MuiTypography-body1-47 MuiPaper-root-11 MuiPaper-elevation6-19 MuiSnackbarContent-root-8 CustomSnackbar-success-1"
  role="alertdialog"
>
  <div
    className="MuiSnackbarContent-message-9"
  >
    <span
      className="CustomSnackbar-message-7"
      id="client-snackbar"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-73 CustomSnackbar-icon-5 CustomSnackbar-iconVariant-6"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
        />
      </svg>
      Meal created successfully
    </span>
  </div>
  <div
    className="MuiSnackbarContent-action-10"
  >
    <button
      aria-label="Close"
      className="MuiButtonBase-root-88 MuiIconButton-root-82 MuiIconButton-colorInherit-83"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiIconButton-label-87"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-73 CustomSnackbar-icon-5"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
          />
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
        </svg>
      </span>
      <span
        className="MuiTouchRipple-root-91"
      />
    </button>
  </div>
</div>
`;
